addons/website_slides_survey/views/website_slides_templates_lesson_fullscreen.xml

```diff
@@ -4,6 +4,7 @@
     <template id="slide_fullscreen_sidebar_category" inherit_id="website_slides.slide_fullscreen_sidebar_category">
         <xpath expr="//li[@t-att-data-id='slide.id']" position="attributes">
             <attribute name="t-att-data-certification-id">slide.survey_id.id</attribute>
+            <attribute name="t-att-data-is-certification">'true' if slide.survey_id.certification else 'false'</attribute>
```

Comment:
Thanks for the PR!

We've had an internal discussion about this and the end-results changes from comments given on Discord.

Basically, this is not worth fixing in 16.0 (the user misconfigured something when untoggling `certification` and could revert to fix).

It would be acceptable in 18.0 though.
In that case we could perhaps be slightly more precise for this condition, so that it checks similarly to what the route will end-up doing (e.g., `user_input` has to exist as well or the button can appear even though the record was deleted).